src/kirby/components/calendar/calendar.component.tns.html

```diff
@@ -0,0 +1,3 @@
+<StackLayout>
+    <WebView #calendarContainer [style.height]="380"></WebView>
```

Comment:
Fixed heights are nasty, but that being said, if this value was 320 instead the look of the calendar would more closely resemble the design shown in Zeplin (with less whitespace between dates and buttons)

Reply:
Fixed